website/docs/language/resources/ephemeral/index.mdx

```diff
@@ -74,11 +74,8 @@ resource "aws_db_instance" "example" {
 }
 ```
 
-In the above example, the ephemeral resource `aws_secretsmanager_secret_version` references an input argument that is initially unknown. As a result, Terraform defers its execution to the apply stage, ensuring that the resource is evaluated at the correct time. 
```

Comment:
Moving this beautiful extra context to sit alongside the example in the write-only article because we want to focus on the high-level of write-only args for this part of the overview, and I feel like this is too indepth.

Reply:
(also moving it out of the `CodeBlockConfig`)